db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
```

Comment:
I'm a bit worried about pushing this change to master. What happened in your schema??

Reply:
Oh no!!!